app/src/main/java/org/p2p/wallet/rpc/RpcModule.kt

```diff
@@ -36,93 +33,59 @@ import org.p2p.wallet.rpc.repository.token.RpcTokenRemoteRepository
 import org.p2p.wallet.rpc.repository.token.RpcTokenRepository
 import org.p2p.wallet.rpc.repository.transaction.RpcTransactionRemoteRepository
 import org.p2p.wallet.rpc.repository.transaction.RpcTransactionRepository
+import retrofit2.Retrofit
 
 object RpcModule : InjectionModule {
 
+    const val RPC_QUALIFIER = "RPC_QUALIFIER"
```

Comment:
hows the idea to add some context to the name?
it's for creating `named` retrofit, right? how about RPC_RETROFIT / RPC_RETROFIT_QUALIFIER?